ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -2894,6 +2894,138 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure 'Audit Credential Validation' is set to 'Success and Failure'
+  platforms: win10
+  platform: windows
+  description: |
+    Creates audit events whenever an attempt is made to authenticate, whether it is successful or
```

Comment:
For formatting, we're trying to not break sentences up between lines which is what happens when we copy and paste from that pdf. So for example, we should combine lines 2902 and 2903 onto a single line.